google-cloud-storage/src/main/java/com/google/cloud/storage/BlobInfo.java

```diff
@@ -882,6 +894,14 @@ public StorageClass getStorageClass() {
     return storageClass;
   }
 
+  /**
+   * Returns the time at which the object's storage class was last changed. When the object is
+   * initially created, it will be set to timeCreated.
+   */
```

Comment:
Returns the time that the object's storage class was last changed or the time of the object creation.

Reply:
done